module_utils/network_lsr/utils.py

```diff
@@ -275,7 +260,7 @@ def mac_aton(mac_str, force_len=None):
     def mac_ntoa(mac):
         if mac is None:
             return None
-        return ":".join(["%02x" % c for c in mac])
+        return ":".join(["%02x" % c for c in bytearray(mac)])
```

Comment:
please mention that bytearray is needed for python2 compatibility.

Reply:
Done, thanks!